apps/champions/lib/champions/config.ex

```diff
@@ -42,9 +53,17 @@ defmodule Champions.Config do
         base_health: Integer.parse(health) |> elem(0),
         base_defense: Integer.parse(defense) |> elem(0),
         game_id: Utils.game_id(),
+        basic_skill_id: get_skill(basic_skill).id,
+        ultimate_skill_id: get_skill(ultimate_skill).id,
         active: true
       }
     end)
     |> Characters.upsert_characters()
   end
+
+  defp get_skill(skill_name) do
```

Comment:
Why do we need this function? Just call Skills.get_skill_by_name(skill_name).
If you're afraid that a Skill with `skill_name` may not exist:
a. you're still not handling it since you don't have an else clause in your with
b. you can just use a default value `Map.get(basic_skill_that_could_be_nil, :id, default_value)`

Reply:
Right, didn't consider that.
In any case, if the skill_name doesn't exist Skills.get_skill_by_name/1 will return nil, which I guess is ok if we don't have a skill in the json for that character.